internal/log/scrub.go

```diff
@@ -172,3 +196,11 @@ func hasKeywords(b []byte) bool {
 	}
 	return false
 }
+
+func trimStringNewline(s string) string {
```

Comment:
Isn't there a golang function in `strings` to do this?

Reply:
there is, I forgot about it